src/components/vertical-divider/vertical-divider.component.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import propTypes from '@styled-system/prop-types';
+import { StyledVerticalWrapper, StyledDivider } from './vertical-divider.style';
+
+const VerticalDivider = ({
+  h,
+  displayInline = false,
+  color = 80,
+  ...props
+}) => (
+  <StyledVerticalWrapper
+    data-component='vertical-divider'
+    pt={ props.pl || 3 }
+    pb={ props.pl || 3 }
+    pr={ props.pr || 3 }
+    pl={ props.pl || 3 }
```

Comment:
Rather than doing each of the 4 paddings like this, it would be cleaner to just do 

```js
p={ props.p || 3}
```

Which will default all paddings to 3, then if the user passes any of the other padding props it'll use that instead

Reply:
The way it works is `p` will be overriden by `px` and `py`, which in turn will be overridden by `pl`, `pr`, `pb` and `pt`

Reply:
Yeah good point I'll update it thanks